lib/Core/ErrorState.cpp

```diff
@@ -329,6 +322,25 @@ ref<Expr> ErrorState::propagateError(Executor *executor,
     valueErrorMap[instr] = result;
     return result;
   }
+  case llvm::Instruction::GetElementPtr: {
+    // Check if base is symbolic and whether a symbolic error value already
+    // exists for the newly pointed element
+    ref<Expr> newError;
+    if (isInStoredError(result)) {
+      newError = retrieveStoredError(result);
```

Comment:
@domainexpert does this mean that the valueErrorMap structure will be retained and not replaced by propagating error through Cells?